File: Task1/C/main.c
#include <stdio.h>

int multipleOf3(){
    int sum = 0;
    //Loop through all numbers from 0 to 1000
    for (int i = 0; i < 1000; i++)
        //Check if multiple of 3 or 5
        if (i % 3  == 0 || i % 5 ==0){
            //Add value of i to sum
            sum += i;
        }
    return sum;
}
int main() {
    printf("The sum of all the multiples of 3 or 5 below 1000 = %i\n",multipleOf3());
    return 0;
}

File: Task1/Python/main.py
def multiple_of_3():
    sum = 0
    for i in range(100):
        if i % 3 == 0 or i % 5 == 0:
            sum = sum + i
    return sum


def main():
    print('The sum of all multiples of 3 or 5 below 1000 = ' + str(multiple_of_3()))


if __name__ == '__main__':
    main()
